js/app.js

```diff
@@ -1,57 +1,19 @@
+/* eslint-disable no-undef */
+
```

Comment:
- [ ] Try not to disable eslint, instead you can fix this by following good code practices.